runtime/stdlib/contracts/test.cdc

```diff
@@ -12,6 +12,103 @@ pub contract Test {
             self.backend = backend
         }
 
+        /// Returns the Flow token balance for the given account.
+        ///
+        pub fun getFlowBalance(_ account: Test.Account): UFix64 {
+            let script = "pub fun main(address: Address): UFix64 { return getAccount(address).balance }"
```

Comment:
Instead of this line:
```cadence
let script = "pub fun main(address: Address): UFix64 { return getAccount(address).balance }"
```
I would have like to be able to use the following:
```cadence
let script = Test.readFile("scripts/get_flow_token_balance.cdc")
```
However, the `Test.readFile` method is implemented natively, and I can't seem to be able to use it from within the `Test` contract.